go/src/koding/klient/machine/index/change.go

```diff
@@ -103,6 +103,25 @@ func (cm *ChangeMeta) Coalesce(newer ChangeMeta) ChangeMeta {
 	}
 }
 
+// String implements fmt.Stringer interface and pretty prints stored change.
+func (cm *ChangeMeta) String() string {
+	cpy := atomic.LoadUint64((*uint64)(cm))
+
+	const cmstr = "LRaduH"
```

Comment:
this is not future proof, if someone adds a new lines betwee iota consts, this would break. Please either use a map/struct with their integer/char representation or if you have any better idea.